doc/man7/fips_module.pod

```diff
@@ -61,7 +61,7 @@ were given during the build process. You can check the location of the config
 file by running this command:
 
     $ openssl version -d
-    OPENSSLDIR: "/usr/local/ssl"
+    $ /usr/local/ssl
```

Comment:
Why this change?

Reply:
because it seemed odd when specifying -d for openssl dir to explicitly print OPENSSLDIR on the output.  I can reinstate that if you like though

Reply:
Please revert that change. People can have scripts that depend on this.